man/systemd-nspawn.xml

```diff
@@ -980,6 +980,20 @@
         effect.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>--notify-mode=</varname></term>
+
+        <listitem><para>Configures support for notifications from init process in the container.
+        <varname>--notify-mode=</varname> supports three options: <option>no</option>, <option>yes</option>, and <option>ready</option>.
+        Option <option>no</option> do not enable notifications between systemd-nspawn and
+        init process in the container and systemd-nspawn notifies systemd with a <literal>READY=1</literal>
+        message when the init process is created. Option <option>yes</option> enables notifications
+        but do nothing with them, systemd-nspawn notifies systemd with a <literal>READY=1</literal>
+        message when the init process is created. Option <option>ready</option>
+        enables notifications and wait for the <literal>READY=1</literal> message from
+        the init process in the container before sending its own to systemd.</para></listitem>
+      </varlistentry>
```

Comment:
I can agree to always listen, but if you do not need the functionality why leave it open than?.
Regarding the boolean, I like the idea that you could implement different behaviours, now there is only "wait for readiness", but it might useful in future to wait for different events.